YMCacheTests/YMMemoryCacheSpec.m

```diff
@@ -4,6 +4,14 @@
 
 #import <YMCache/YMCache.h>
 
+void pumpRunLoop(NSTimeInterval forSeconds) {
+    NSDate *stopDate = [NSDate dateWithTimeIntervalSinceNow:forSeconds];
+    NSTimeInterval stopInterval = stopDate.timeIntervalSinceReferenceDate;
+    while (stopInterval >= [NSDate date].timeIntervalSinceReferenceDate) {
+        [[NSRunLoop currentRunLoop] runUntilDate:stopDate];
+    }
+}
+
 SpecBegin(YMMemoryCacheSpec)
 
 fdescribe(@"YMMemoryCache", ^{
```

Comment:
This should not be an `fdescribe`. Will bite somebody who add's the next spec outside of this scope.